RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -266,12 +266,16 @@ void PixelThresholdClusterizer::copy_to_buffer( DigiIterator begin, DigiIterator
   for(DigiIterator di = begin; di != end; ++di) {
     int row = di->row();
     int col = di->column();
-    int adc = electron[i++];
+    int adc = electron[i++]; // this is in electrons 
+
 #ifdef PIXELREGRESSION
     int adcOld = calibrate(di->adc(),col,row);
     //assert(adc==adcOld);
     if (adc!=adcOld) std::cout << "VI " << eqD  <<' '<< ic  <<' '<< end-begin <<' '<< i <<' '<< di->adc() <<' ' << adc <<' '<< adcOld << std::endl; else ++eqD;
 #endif
+
+    if(adc<100) adc=100; // put all negative pixel charges into the 100 elec bin 
```

Comment:
since the comment is about "negative pixel charges", why `if(adc<100)` rather than `if(adc<0)` ?

Reply:
wouldn't it make more sense to have the cut value (`adc<100`) and target value (`adc=100`) independent and configurable ?